src/lib/evas/canvas/meson.build

```
@@ -1,7 +1,49 @@
+pub_eo_files_optional_ector_ = [
+  ['efl_canvas_group.eo', 'efl_canvas_group_vg.eo_template', 'efl_canvas_group_non_vg.eo_template'],
+  ['efl_canvas_image_internal.eo', 'efl_canvas_image_internal_vg.eo_template', 'efl_canvas_image_internal_non_vg.eo_template'],
+  ['efl_canvas_textblock.eo', 'efl_canvas_textblock_vg.eo_template', 'efl_canvas_textblock_non_vg.eo_template'],
```

Not too much, but...

```suggestion
  ['efl_canvas_group.eo',
   'efl_canvas_group_vg.eo_template',
   'efl_canvas_group_non_vg.eo_template'],
  ['efl_canvas_image_internal.eo',
   'efl_canvas_image_internal_vg.eo_template',
   'efl_canvas_image_internal_non_vg.eo_template'],
  ['efl_canvas_textblock.eo',
   'efl_canvas_textblock_vg.eo_template',
   'efl_canvas_textblock_non_vg.eo_template'],
```

We could also change it into a dict (for better semantics) or just store the prefix and add `_vg` and `_non_vg` + `.eo` and `.eo_template` inside the `foreach` loop.

---

I've reworked this, what do you think about this?